app/assets/javascripts/ui_models/application.ts

```diff
@@ -25,6 +25,8 @@ import { NativeExtManager } from '@/services/nativeExtManager';
 import { StatusManager } from '@/services/statusManager';
 import { ThemeManager } from '@/services/themeManager';
 
+declare const __VERSION__: string;
```

Comment:
Can this be imported from somewhere instead? Perhaps we should have a version.ts file that all other files reference if they need to access the version?

Reply:
Nice point, moved this and other globals to separate files. Actually my intention was to combine them in one file end export, but only default export works in this case, so created files for every global item.